File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/MainAppWebResponseContext.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class MainAppWebResponseContext{

	@SerializedName("loggedOut")
	private boolean loggedOut;

	public void setLoggedOut(boolean loggedOut){
		this.loggedOut = loggedOut;
	}

	public boolean isLoggedOut(){
		return loggedOut;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/MainAppWebResponseContext.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class MainAppWebResponseContext : Parcelable, Serializable {
    @SerializedName("loggedOut")
    var isLoggedOut = false
    override fun toString(): String {
        return "MainAppWebResponseContext{" +
                "loggedOut = '" + isLoggedOut + '\'' +
                "}"
    }
}